Sources/message/ALMessage.m

```diff
@@ -329,7 +329,7 @@ - (BOOL)isPushNotificationMessage {
 
 - (BOOL)isMessageCategoryHidden {
     return (self.metadata && [self.metadata valueForKey:@"category"] &&
-            [[self.metadata valueForKey:@"category"] isEqualToString:AL_CATEGORY_HIDDEN]);
+            [[self.metadata valueForKey:@"category"] isEqualToString:AL_CATEGORY_HIDDEN] && self.contentType != 0);
```

Comment:
Are we adding this check to hide the message in sdk side ?

Reply:
Added a new check to hide the "KM_EVENT_QUICKREPLY_CLICK" for the end user